master/getting-started/kubernetes/installation/config-options.md

```diff
@@ -67,23 +67,79 @@ The following table outlines the supported `ConfigMap` options for etcd:
 
 | Option                 | Description    | Default
 |------------------------|----------------|----------
-| etcd_endpoints         | A comma separated list of etcd nodes. | http://127.0.0.1:2379
-| etcd_ca                | The location of the CA mounted in the pods deployed by the DaemonSet. | None
-| etcd_key               | The location of the client cert mounted in the pods deployed by the DaemonSet. | None
-| etcd_cert              | The location of the client key mounted in the pods deployed by the DaemonSet. | None
+| etcd_endpoints         | Comma-delimited list of etcd endpoints to connect to. | http://127.0.0.1:2379
+| etcd_ca                | The root certificate of the CA that issued the etcd server certificate. Configures `{{site.nodecontainer}}`, the CNI plugin, and the Kubernetes controllers to trust the signature on the certificates provided by the etcd server. | None
+| etcd_key               | The private key of the `{{site.nodecontainer}}`, the CNI plugin, and the Kubernetes controllers client certificate. Enables these components to participate in mutual TLS authentication and identify themselves to the etcd server. | None
+| etcd_cert              | The client certificate issued to `{{site.nodecontainer}}`, the CNI plugin, and the Kubernetes controllers. Enables these components to participate in mutual TLS authentication and identify themselves to the etcd server. | None
 
 To use these manifests with a TLS-enabled etcd cluster you must do the following:
 
-- Populate the `calico-etcd-secrets` secret with the contents of the following files:
-  - `etcd-ca`
-  - `etcd-key`
-  - `etcd-cert`
-
-- Populate the following options in the `ConfigMap` which will trigger the various
-  services to expect the provided TLS assets:
-  - `etcd_ca: /calico-secrets/etcd-ca`
-  - `etcd_key: /calico-secrets/etcd-key`
-  - `etcd_cert: /calico-secrets/etcd-cert`
+1. Download the {{page.version}} manifest that corresponds to your installation method.
+
+   **{{site.prodname}} for policy and networking**
+   ```bash
+   curl \
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/calico.yaml \
+   -O
+   ```
+
+   **{{site.prodname}} for policy and flannel for networking**
```

Comment:
We should get rid of the `canal-etcd.yaml` manifest - it's not a recommended configuration.